src/firewall/core/rich.py

```diff
@@ -471,8 +473,35 @@ def value_normalized(self, check=True):
             return self.value
         return f"{rate}/{duration}"
 
+    def burst_parse(self):
+        if self.burst is None:
+            return None
+        try:
+            burst = int(self.burst)
+        except:
+            raise FirewallError(errors.INVALID_LIMIT, self.burst)
+
+        if burst < 1 or burst > 10_000_000:
+            raise FirewallError(errors.INVALID_LIMIT, self.burst)
+
+        return burst
+
+    def burst_normalized(self, check=False):
+        try:
+            burst = self.burst_parse()
+        except FirewallError:
+            if check:
+                raise
+            return self.burst
+        if burst is None:
+            return None
+        return f"{burst}"
+
```

Comment:
Just set `self.burst` to a normalized value when the object is created. No need for these functions.

Reply:
the need is to have stateless data holders, that can contain the input values of the caller (without normalization).

Reply:
changed.